cloudlift/deployment/service_template_generator.py

```diff
@@ -959,6 +981,71 @@ def _get_desired_task_count_for_service(self, service_name):
             return self.desired_counts[service_name]
         else:
             return 0
+    def _firelens_container_def_args_override(self, configuration, container_def_args):
+        if configuration.get('logging') == 'awsfirelens':
+            container_def_args['DependsOn'] = container_def_args.get('DependsOn', []) + [ContainerDependency(ContainerName='fluentbit-firelens-sidecar', Condition='START')]
+        return container_def_args
+
+    def _sidecar_container_defs(self, configuration, service_name):
+        container_definitions = []
+        if configuration.get('sidecars') and isinstance(configuration['sidecars'], list):
+            for index, sidecar in enumerate(configuration['sidecars']):
+                container_name = sidecar.get('name', f"sidecar_{index + 1}")
+                if not container_name.endswith('-sidecar'):
+                    container_name = container_name + '-sidecar'
+                image_uri = sidecar.get('image_uri')
+                memory_reservation = int(sidecar.get('memory_reservation', 50))
+                essential = sidecar.get('essential', True)
+                
+                mount_points = []
+                if sidecar.get('mount_points') and isinstance(sidecar['mount_points'], list):
+                    for mount_point in sidecar['mount_points']:
+                        container_path = mount_point.get('container_path')
+                        source_volume = mount_point.get('source_volume')
+                        if container_path and source_volume:
+                            mount_points.append(MountPoint(ContainerPath=container_path, SourceVolume=source_volume))
+                
+                container_def_args = {}
+                command = sidecar.get('command')
+                if command is not None:
+                    container_def_args['Command'] = [command]
+                    
+                env = sidecar.get('env')
+                if env is not None and isinstance(env, dict):
+                    container_def_args['Environment'] = []
+                    for key, value in env.items():
+                        container_def_args['Environment'].append(Environment(Name=key, Value=value))
+                logging = sidecar.get('logging')
+                if logging is not None:
+                    container_def_args['LogConfiguration'] = self._gen_log_config(service_name + f"-{container_name}", "awslogs" if 'logging' not in sidecar else logging)
```

Comment:
By default sidecar container , should have logging as awsfirelens, unless it is a fluentbit sidecar @rokibulislaam

Reply:
default has been set to `awslogs` for now